src/Symfony/Component/Dsn/Factory/MemcachedConnectionFactory.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Dsn\Factory;
+
+use Symfony\Component\Dsn\Exception\InvalidArgumentException;
+
+/**
+ * Factory for Memcached connections.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
```

Comment:
it's new code. is he an author here indeed?

Reply:
this is code extracted from the Cache component, so he is.

Reply:
cool. then we shall use this in Cache component ;)

Reply:
please read the PR description. It says it is the next step.